tests/mod.rs

```diff
@@ -272,6 +273,27 @@ fn cargo_home_filtering() {
 }
 
 #[test]
+fn rustflags_handling() {
+    check_percentage("rustflags", 1.0f64, true);
+    env::set_var("RUSTFLAGS", "--cfg=foo");
+    let mut config = Config::default();
+
+    let restore_dir = env::current_dir().unwrap();
+    let test_dir = get_test_path("rustflags");
+    env::set_current_dir(&test_dir).unwrap();
+    config.manifest = test_dir;
+    config.manifest.push("Cargo.toml");
+
+    let (_, ret) = launch_tarpaulin(&config, &None).unwrap();
+    env::set_current_dir(restore_dir).unwrap();
```

Comment:
ah sorry I've lead you astray! This line should be deleted as you restore the directory later